agent/command/git.go

```diff
@@ -421,7 +422,11 @@ func (c *gitFetchProject) Execute(ctx context.Context, comm client.Communicator,
 		return errors.Wrap(err, "error expanding github parameters")
 	}
 
-	projectMethod, projectToken, err := getProjectMethodAndToken(c.Token, conf.Expansions.Get(evergreen.GlobalGitHubTokenExpansion), conf.Distro.CloneMethod)
+	cloneMethod := distro.CloneMethodOAuth
```

Comment:
Since it's no longer just a property of distros, can we move `distro.CloneMethodOAuth` and `distro.CloneMethodSSH` into the global constants in the top-level `evergreen` package?

Reply:
Done